workspaces/homepage/plugins/dynamic-home-page/src/components/EntitySection/CardWrapper.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { Link } from '@backstage/core-components';
+
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+
+import TagList from './TagList';
+
+interface CardWrapperProps {
+  link: string;
+  title: string;
+  version?: string;
+  description: string;
+  tags: string[];
+}
+
+const CardWrapper: React.FC<CardWrapperProps> = ({
+  link,
+  title,
+  description,
+  tags,
+}) => {
+  return (
+    <Box
+      sx={{
+        border: theme => `1px solid ${theme.palette.grey[400]}`,
+        borderRadius: 3,
+        overflow: 'hidden',
+      }}
+    >
+      <CardContent
+        sx={{
+          pb: 2,
+          '&:last-child': {
+            pb: 2,
+          },
+          backgroundColor: 'transparent',
+        }}
+      >
+        <Box sx={{ overflow: 'hidden' }}>
+          <Link
+            to={link}
+            underline="always"
+            style={{
+              display: '-webkit-box',
+              WebkitBoxOrient: 'vertical',
+              WebkitLineClamp: 1,
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+              fontSize: '0.9rem',
+              fontWeight: '500',
+            }}
+          >
+            {title}
+          </Link>
+        </Box>
+
+        {/* // Todo: Add version to the model card once it is available
+        <Box sx={{ display: 'flex', alignItems: 'center', pt: 1 }}>
+          <LocalOfferOutlinedIcon sx={{ width: '16px' }} />
+          <Typography variant="body2">{version}</Typography>
+        </Box> */}
+        <Box sx={{ pt: 2, height: '175px', overflow: 'hidden' }}>
+          <Typography
+            variant="body2"
+            paragraph
+            sx={{
+              display: '-webkit-box',
+              WebkitBoxOrient: 'vertical',
+              WebkitLineClamp: 8,
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+            }}
+          >
+            {description}
+          </Typography>
+        </Box>
+        <TagList tags={tags} />
```

Comment:
In the [design](https://www.figma.com/design/6ORS1mhUaxeyTBjMmE95Rn/Home-page-redesign?node-id=1452-1757&t=wigM7YCFmStp7pcM-0), every card starts with coloured badge denoting the `kind` of the entity.

Design:
![image](https://github.com/user-attachments/assets/b27db6d1-0595-4378-9edb-7ff1e6615100)


 Can we show the `kind` as the first badge and then followed by tags (if available)? Currently we can't say which type of entity is showing up.

![image](https://github.com/user-attachments/assets/806d7eda-3e8f-421d-9a93-00e2d223b9cc)